src/studio/src/designer/backend/Controllers/DatamodelsController.cs

```diff
@@ -46,6 +47,7 @@ public DatamodelsController(IRepository repository)
         public async Task<IActionResult> UpdateDatamodel(string org, string app, string filepath)
         {
             SchemaKeywordCatalog.Add<InfoKeyword>();
+            filepath = filepath.AsFileName();
```

Comment:
filepath contains typically App%2Fmodels%2Fdefault. Using AsFileName would break the existing implementation.

Reply:
my bad. Thought it would be a quick fix.